src/components/schemalist/schemalistitem.js

```diff
@@ -44,18 +44,19 @@ angular.module('vlui')
         // TODO(https://github.com/vega/vega-lite-ui/issues/187):
         // consider if we can use validator / cql instead
         var allowedCasting = {
-          quantitative: [vl.type.QUANTITATIVE, vl.type.ORDINAL, vl.type.NOMINAL],
-          ordinal: [vl.type.ORDINAL, vl.type.NOMINAL],
-          nominal: [vl.type.NOMINAL, vl.type.ORDINAL],
-          temporal: [vl.type.TEMPORAL, vl.type.ORDINAL, vl.type.NOMINAL],
+          integer: [vl.type.QUANTITATIVE, vl.type.ORDINAL, vl.type.NOMINAL],
+          number: [vl.type.QUANTITATIVE, vl.type.ORDINAL, vl.type.NOMINAL],
+          date: [vl.TEMPORAL, vl.type.NOMINAL, vl.type.ORDINAL],
```

Comment:
I know that previously I say date could be ordinal / nominal.  But I think that's wrong.  I think it should be just temporal.  What do you think?